andromeda-ui/components/ObservationTable.tsx

```diff
@@ -22,15 +23,13 @@ const KNOWN_COLUMNS = new Set([
 ]);
 
 export default function ObservationTable(props: ObservationTableProps) {
-    const { observations, maxObs, iNatUser } = props;
+    const { observations, maxObs, iNatUser, totalObservations } = props;
     const showing = Math.min(observations.length, maxObs);
     let extraColumns: string[] = [];
     if (observations.length > 0) {
         extraColumns = Object.keys(observations[0]).filter((x) => !KNOWN_COLUMNS.has(x))
     }
-    // since the default order of observations is date ascending reverse
-    // show we are showing the most recent observations
-    const exampleObservations = observations.slice().reverse().slice(0, maxObs);
+    const exampleObservations = observations.slice().slice(0, maxObs);
```

Comment:
Did the order of `observations` change that the `reverse` is no longer necessary?

Reply:
Yes. The `pyinaturalist.get_observations()` function orders the most recent first when called with a limit (`per_page`).
https://github.com/Imageomics/Andromeda/blob/de72f2103d9ac777888fdb0de8c0818d6dcaa059/andromeda/inaturalist.py#L63-L68
I found this rather surprising design of get_observations.